com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -2,7 +2,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-namespace Unity.Netcode.Prototyping
+namespace Unity.Netcode.Components
```

Comment:
Does having this namespace buy us anything?  I like moving things out of the Prototyping folder.  But like, we have a number of things that are in folders (e.g. NetworkVariable) that are not in a `NetworkVariable` different namespace.

Reply:
I'm going to inline some parts of our Slack conversation here too:

> I think the difference is NetworkTransform, NetworkAnimator etc are going to be bespoke Unity-tailored components
> Components is more like "bespoke network components to use in Unity world"
> So, I wouldn't say NetworkVariable will be kicked out of Runtime/Core in the future, we'll probably keep it in the core long term